### PR TITLE
Try to fix coveralls

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

**Comment by nomis52** [comment]
Alright, now that I've figured out how this all works, I realized I missed the second os.walk in the cpp-coveralls script.

**Comment by nomis52** [comment]
I've realized that I missed the second os.walk in the coveralls script.

**Comment by peternewman** [comment]
There are two more os.walk calls @nomis52 /

**Comment by nomis52** [comment]
PR sent.

File: .travis.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ before_install:
 
 after_success:
  - if [ "$TASK" = "coverity" ]; then cat /home/travis/build/${TRAVIS_REPO_SLUG}/tests/cov-int/scm_log.txt; fi
- - if [ "$TASK" = "coverage" ]; then cd tests && coveralls --gcov /usr/bin/gcov-5 -b . -e include -e mocks -e config.h -e system_config -E '.*Test\.cpp$' --gcov-options '\-lp' > /dev/null; fi
+ - if [ "$TASK" = "coverage" ]; then cd tests; coveralls --gcov /usr/bin/gcov-5 -b . -e include -e mocks -e config.h -e harmony -e system_config -E '.*Test\.cpp$' --gcov-options '\-lp' --follow-symlinks > /dev/null; fi
```
Comment (peternewman) [inline .travis.yml]: We could revert this to &&

Comment (nomis52) [inline .travis.yml]: I don't think it matters much. Feel free to change.

**Pull request merged**